Cargo.lock

```diff
@@ -407,7 +407,7 @@ dependencies = [
  "heck",
  "proc-macro2",
  "quote",
- "syn",
+ "syn 2.0.79",
```

Comment:
Hmm, that's unfortunate that dir-test still uses `syn` one but so be it. We can always make a PR upstream